go.mod

```diff
@@ -0,0 +1,11 @@
+module github.com/editorconfig/editorconfig-core-go
```

Comment:
This patch will break stuff for current users unless we bump the version to v2 and tag this as v2.0.0.

```suggestion
module github.com/editorconfig/editorconfig-core-go/v2
```

Reply:
Correct!

Reply:
I think we should do that.
There is nothing better to come than go modules in the near future.